packages/client/webpack.config.js

```diff
@@ -113,13 +113,16 @@ module.exports = (env) => {
   return {
     context: path.resolve(process.cwd()),
     mode: configs[currentEnv].mode,
-    devtool: configs[currentEnv].devtool,
+    devtool: configs[currentEnv].devtool == 'true',
```

Comment:
We can change line 105 to `none` and remove arguments from `package.json`

ref: https://webpack.js.org/configuration/devtool/